src/test/java/uk/gov/justice/digital/hmpps/keyworker/integration/KeyworkerSearchIntegrationTest.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.keyworker.integration
+
+class KeyworkerSearchIntegrationTest : IntegrationTest() {
+  companion object {
+    const val PRISON_ID = "LEI"
+    const val USERNAME = "Another"
+  }
+
+  val STAFF_DETAILS = getWiremockResponse(PRISON_ID, "staff-location-role-list")
+  val CASE_NOTE_USAGE = getWiremockResponse("case-note-usage")
+
+  fun `keyworker search - decorated with defaults after migration`() {
```

Comment:
Skipping for now. Will pick it back up later.

Reply:
You could annotate it with @Disabled with some explanation text - useful as a reminder.

Reply:
I'll add that now locally, thanks Mike